tools/eosiocpp.in

```diff
@@ -112,6 +112,7 @@ function generate_abi {
     ${ABIGEN} -extra-arg=-c -extra-arg=--std=c++14 -extra-arg=--target=wasm32 \
         -extra-arg=-nostdinc -extra-arg=-nostdinc++ -extra-arg=-DABIGEN \
         -extra-arg=-I@CMAKE_SOURCE_DIR@/contracts \
+        -extra-arg=-I@CMAKE_SOURCE_DIR@/build/contracts \
```

Comment:
No adding an installed path won't work.

You may try this command yourself:

    $ sudo find / | grep eosiolib/core_symbol.hpp

And you'll see that the file is *not* included in the installation directory after a normal install. The `eosiolib` files used to be there, but they've been removed since the deprecation.

Reply:
@lazaridiscom I agree though that `@CMAKE_SOURCE_DIR@/build` should be replaced with `@CMAKE_BINARY_DIR@`. I didn't know this variable existed, but it's already been fixed by an [additional commit](https://github.com/EOSIO/eos/pull/5401/commits/4655c8f708a23c560c6ee5f94905c2cf5016d6ec) from @larryk85

Reply:
And you may check out the 66th line of the file `eosiocpp.in`:

    -I @CMAKE_BINARY_DIR@/contracts \

I'm just adding the same include path to `ABIGEN`. It's incorrect to try to get the `eosiolib` into the install directory as the tool is being deprecated. It's now in the `eosio.wasmsdk` install directory instead.

Reply:
I still believe that using "build/" within include-paths is invalid, because the `build` is essentially a temporary directory.

But your logic to copy the existent construct is of course perfectly valid.